app/src/main/java/org/p2p/wallet/home/ui/main/HomePresenter.kt

```diff
@@ -107,10 +112,25 @@ class HomePresenter(
     }
 
     override fun onBuyTokenClicked(token: Token) {
+        if (token.tokenSymbol !in TOKENS_VALID_FOR_BUY) {
+            view?.showBuyInfoScreen(token)
+        } else {
+            onBuyToken(token)
+        }
+    }
+
+    override fun onInfoBuyTokenClicked(token: Token) = onBuyToken(token)
```

Comment:
block body

Reply:
:/ we need to think about not use block when case was just redirected ... 3 lines except of 1 line... wtf ?